examples/chip-tool/README.md

```diff
@@ -579,3 +579,25 @@ To learn more about the tool, how to build it, use its commands and advanced
 features, read the following guide:
 
 -   [Working with the CHIP Tool](https://github.com/project-chip/connectedhomeip/tree/master/docs/guides/chip_tool_guide.md)
+
+## Using the Client via PASE
```

Comment:
PASE is only for commissioning. While the commissioning protocol makes use of commands over PASE, and indeed, anything can be done against the device during commissioning, you cannot rely on PASE forever: on PASE establishment, a fail-safe timer will be set, which will cause eventual reset of state. Also, the commissioning window can only be open for 15 minutes at a time. Could you please explain why this new change is encouraging *operational* use of PASE over BLE for non-commissioning use-cases? Illustrating this without the proper warnings may cause SDK users to believe they can use PASE over BLE "forever" to control devices operationally, which is explicitly a problem by spec (due to aforementioned commissioning window behavior).

Reply:
I spoke with Marcus about this on slack - they would like to use the pase connection to get ID information about the device before commissioning. This is appropriate for such a use case. Adding this in the documentation would be beneficial.

Reply:
@tcarmelveilleux 

it's understood that PASE is not to be used for true operational use-cases. however, i see several uses:
* usage of PASE for end-of-line production tests with manufacturer specific clusters
* usage of PASE for identifying the device while commissioning

i will add the disclaimers to the README.